src/features/social/Feed.tsx

```diff
@@ -457,7 +458,7 @@ const FeedContent: React.FC<FeedContentProps> = ({
   const scrollContainerRef = useRef<HTMLDivElement>(null);
   const loaderRef = useRef<HTMLDivElement>(null);
   const { t } = useAppTranslation();
-
+  const now = useNow({ live: true });
```

Comment:
Not sure we want this to be live at this level. It will cause a re render of a lot for a very small component. Again should probably be moved to a separate component with now inside it.